scripts/sql/metadata_schema.sql

```diff
@@ -341,7 +341,7 @@ CREATE TABLE IF NOT EXISTS `pixels_metadata`.`FILES` (
 -- Table `pixels_metadata`.`SINGLE_POINT_INDICES`
 -- -----------------------------------------------------
 CREATE TABLE IF NOT EXISTS `pixels_metadata`.`SINGLE_POINT_INDICES` (
-    `SPI_ID` BIGINT NOT NULL,
```

Comment:
We should modify pixels_metadata.mwb under pixels-daemon/src/main/resources/ accordingly.